### DISCUSSION TITLE
Citrix Workspace

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#43] [General]
# Citrix Workspace

Check if Citrix Workspace is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/ca757a18-20b9-4c32-a362-c7febe731278

**Comment by josheliason88** [comment]
I was able to launch Citrix apps from our environment (we don't leverage App Protection) by installing the Citrix Workspace app from the Windows Store.

**Comment by BuenyA** [comment]
I also tested Citrix on my Lenovo Yoga 7x, and it worked well without any issues.

**Reply by OpenSource03** [reply]
Is it running under emulation or native?

**Reply by BuenyA** [reply]
It's running under emulation

**Comment by skyried1** [comment]
Have someone experience with MS Teams on a Citrix Session. Works offloading?

**Reply by viveksh1** [reply]
Yes, any application on Citrix should work

**Comment by viveksh1** [comment]
[According to Citrix](https://www.citrix.com/blogs/2021/03/02/leverage-citrix-workspace-app-for-windows-on-arm64-devices/#:~:text=Citrix%20Workspace%20app%20for%20Windows%20now%20runs%20in%20an%20emulator%20mode%20on%20ARM64%20devices) this should work.

However, the App Protection feature doesn't work under emulation, but usually the App Protection feature is not used so please update this program to "Emulated" status with an asterisk/footnote.

More info: https://community.citrix.com/forums/topic/250915-app-protection-component-windows-11-22h2-seems-not-installed/#_=_

**Comment by ronaldancaya-prog** [comment]
how is the performance?, is it worth it?

**Reply by skyried1** [reply]
same performance like x86. I use it since more as a year and have no problems or performance issues. ..... or not more as x86 users :-)
